harness/determined/cli/experiment.py

```diff
@@ -527,25 +530,33 @@ def kill_experiment(args: Namespace) -> None:
     print("Killed experiment {}".format(args.experiment_id))
 
 
-@authentication.required
-def wait(args: Namespace) -> None:
+def _wait(session: api.Session, experiment_id: int, polling_interval: int) -> None:
+    retry = urllib3.util.retry.Retry(
+        raise_on_status=False,
+        total=10,
+        status=10,
+        backoff_factor=2,
```

Comment:
probably a lower backoff value. open to suggestions on these parameters.

Reply:
also the status force_list.

TODO comment what each one is. i frequently mis label them.